examples/Mediawiki/SiteStatistics.yaml

```diff
@@ -0,0 +1,60 @@
+suite: Site Statisitics
+description: Testing site statistics' edits value
+type: RPC
+
+use-fixtures:
+  - Alice
+
+tests:
+  - description: Edit value should increase in site statistics after editing a page
+    default-session: Alice
+    retry:
+      tries: 3
+    interaction:
+      # Get site statistics
+      - request:
+          path: api.php
+          parameters:
+            action: query
+            meta: siteinfo
+            siprop: statistics
+            format: json
+        response:
+          body:
+            query:
+              statistics:
+                edits: p/grab: edits-stats
+      # Create page
+      - request:
+          method: post
+          path: api.php
+          parameters:
+            action: edit
+            title: Site Statistics
+#            createonly: true
+            format: json
+          form-data:
+            summary: create page
+            text: Will site statistics increase?
+            token: p/var: edit-token
+        response:
+          body:
+            edit:
+              result: Success
+      # Get site statistics
+      - request:
+          path: api.php
+          parameters:
+            action: query
+            meta: siteinfo
+            siprop: statistics
+            format: json
+        response:
+          body:
+            query:
+              statistics:
+                # p/greater: function that checks to see if the current number is greater than the given number
+                #
+                # What happens if a parallel test undos/reverts edits? Should it be greater than or equal?
+                # Might be a good candidate for retry. I've added retry to the entire sequence.
+                edits: p/greater: p/var: edits-stats
```

Comment:
Why do you think the entire sequence should be retried?

Also, we can't really chain tags like that, can we?... But we'd have to have a way to reference variables somehow. Maybe we need p/ax, for "arithmetic expression", that allows us to do something like p/ax: "> edit-stats"?.. but things get rather complicated there...

Can tags be appleid to structures? E.g. p/greater: { value: p/var edit-stats }?

Maybe this just isn't possible within the paradigm? Should be flagged as a problematic case to look at.

Reply:
> Why do you think the entire sequence should be retried?

If for some reason a page is deleted in a parallel test does the edit number go down? Or some other behavior that causes the edit number to decrease in the sequence.

Reply:
No, to the contrary: it wouls go up, because deletion also counts as an edit in these stats. The relevant code can be found in WikiPage::doDeleteUpdates.

Reply:
Good to know! I've updated the test.